lib/opcodes.js

```diff
@@ -175,5 +175,5 @@ module.exports = function (op, full) {
     }
   }
 
-  return {name: opcode, fee: code[1], in: code[2], out: code[3], dynamic: code[4]}
+  return {name: opcode, fee: code[1], in: code[2], out: code[3], dynamic: code[4], async: code[5]}
```

Comment:
note the async column above on line 3 too (`// name, baseCost, off stack, on stack, dynamic, async`)?